### PR TITLE
DMP-402: Implement playback operation

### PR DESCRIPTION
**Pull request by chrisbellingham-hmcts**
[DMP-402](https://tools.hmcts.net/jira/browse/DMP-402)

Implement audio playback functionality. This largely consists of calling existing lower-level functionality such that the requested audio is:
* concatenated
* merged
* trimmed
* re-encoded
* and stored into outbound storage

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by karen-hedges — APPROVED** [review]

**Review by hemantasharma1129 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImpl.java [hunk 1]
```diff
@@ -136,20 +151,35 @@ private List<AudioFileInfo> concatenateByChannel(List<AudioFileInfo> audioFileIn
...
+
+        return audioOperationService.merge(audioFileInfos, StringUtils.EMPTY);
+    }
+
+    private AudioFileInfo trimToPeriod(AudioFileInfo audioFileInfo, OffsetDateTime start,
+                                       OffsetDateTime end) throws ExecutionException, InterruptedException {
+        return audioOperationService.trim(
+            StringUtils.EMPTY,
+            audioFileInfo,
+            start.toLocalTime().toString(),
```
Comment (richardbull) [inline src/main/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImpl.java:165]: I'm not sure this conversion to LocalTime will work here, as the `Application` sets the Default Timezone to be UTC.  The darts-api.jar also sets the system property `-Duser.timezone=UTC` in the Dockerfile.

Also the `trim` startTime, endTime strings are simply the start, stop position of the media using the ffmpeg Time duration in `HH:MM:SS` format (http://ffmpeg.org/ffmpeg-utils.html#Time-duration)

Comment (chrisbellingham-hmcts) [reply src/main/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImpl.java]: Good spot, have updated. Please can you re-review this area? If the approach looks reasonable, would it be better to move the `OutboundFileProcessorImpl.toTimeString(Duration duration)` method into `AudioOperationService` do you think?

Comment (chrisbellingham-hmcts) [reply src/main/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImpl.java]: Discussed with @richardbull and we can leave `toTimeString` here. If other usages in the future will benefit from it being in `AudioOperationService`, we can easily move it.

Comment (richardbull) [reply src/main/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImpl.java]: I think it looks ok what you've done in `OutboundFileProcessorImpl#trimToPeriod`
were you thinking you would change the service in the `AudioOperationService` to overload the trim method to use `Duration `params instead?

Up to you, whether you feel it's worth the hassle - not sure about moving the toTimeString thing - maybe useful as a utility method perhaps

**Review by richardbull — APPROVED** [review]

**Pull request merged**